modules/HapticGlove_module/src/RobotSkin.cpp

```diff
@@ -142,6 +130,100 @@ bool RobotSkin::configure(const yarp::os::Searchable& config,
         return false;
     }
 
+    const bool connectToCalibratedSkin
+        = config.check("connect_to_calibrated_skin", yarp::os::Value(0)).asBool();
```

Comment:
Do you think this could work?
```suggestion
        = config.check("connect_to_calibrated_skin", yarp::os::Value(false)).asBool();
```

Reply:
yes it should work